assets/js/vue/components/data-land/DownloadZipBtn.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+    <div>
+        <b-button size="sm" class="mb-2" @click="downloadBtn" variant="primary">
+            <b-icon icon="arrow-up-right" aria-hidden="true" v-if="datasetInfo.remotelyHosted"></b-icon>
+            <b-icon icon="download" aria-hidden="true" v-else></b-icon>
+            {{ buttonTitle }}
+        </b-button>
+        <DxPopup
+            :title="buttonTitle"
+            :visible.sync="showDownloadDialog"
+            :close-on-outside-click="true"
+            :show-title="true"
+            :width="500"
+            :height="500">
+            <template>
+                <div v-if="datasetInfo.remotelyHosted">
+                    <h3>The dataset you selected is hosted by an external repository.</h3>
+                    <div>
+                        <p>
+                            All materials on this website are made available to GRIIDC and in turn to you "as-is."
+                            By downloading files, you agree to the <a
+                            href=https://data.gulfresearchinitiative.org/terms-and-conditions>GRIIDC Terms of
+                            Service</a>.
+                        </p>
+                        <p>
+                            This particular dataset is not hosted directly by GRIIDC, so additional terms and conditions
+                            may be
+                            imposed by the hosting entity.
+                        </p>
+                    </div>
+                    <div>
+                        <strong>UDI:</strong> {{ datasetInfo.dataset.udi }}<br>
```

Comment:
I'm curious if we should include the DOI in here as well...might be worth asking PO.

Reply:
This is the download button part, the DOI is on the page.